### DISCUSSION TITLE
ZeppOS 2.X on T-Rex 2?

### DISCUSSION THREAD

**Discussion by Antrone-5545** [zepp-health/zeppos-forum#228] [Q&A] [answered]
According to this website, T-rex 2 should receive an update to ZeppOS 2.0 in August. It's true?
https://gadgetsandwearables.com/2023/07/26/amazfit-t-rex-2-zepp-os-2-0/

**Comment by silver-zepp** [comment, marked as answer]
That is correct @Antrone-5545. Send an email to Jin jin@amazfit.com with your Zepp ID. Also, consider joining our community on [Slack](https://t.zepp.com/t/zepp-community-slack) for quicker updates.

**Reply by Antrone-5545** [reply]
🎉 THANK YOU ZEPP DEVELOPERS ❤️

**Reply by hpcolus** [reply]
I have not been able to join the slack community as it says I must have a valid zepp.com email.  How can I join?

**Reply by silver-zepp** [reply]
I see, we'll look into this. Are you able to join with a Google account? @hpcolus

**Reply by silver-zepp** [reply]
Try this link @hpcolus [Join Zepp's Slack](https://join.slack.com/t/zeppcommunity/shared_invite/zt-1xrn69hec-Z5YESagaNJj7g3V6m6w6oQ). You should be able to join with any email.

**Reply by hpcolus** [reply]
Thanks!  Now I am in
